src/AutoMapper/IMappingExpression.cs

```diff
@@ -252,6 +260,21 @@ public interface IMappingExpression<TSource, TDestination>
         IMappingExpression<TSource, TDestination> IgnoreAllSourcePropertiesWithAnInaccessibleSetter();
 
         /// <summary>
+        /// Ignores <typeparamref name="TDestination"/> members as specified in an array of expressions
+        /// </summary>
+        /// <param name="memberExpressionTrees">An array of expressions that evaluate to properties that should be ignored when mapping.</param>
+        /// <returns>Itself</returns>
+        /// <remarks>This method is syntactic sugar for calling .ForMember and .Ignore repeatedly.</remarks>
+        /// <example>
+        /// To ignore several properties when mapping, specify each in a lambda expression.
+        /// <code>
+        /// config.CreateMap&lt;Foo, FooDto&gt;()
+        ///       .IgnoreMembers(target => target.Number, target => target.Name, target => target.CreateDate);
+        /// </code> 
+        /// </example> 
+        IMappingExpression<TSource, TDestination> IgnoreMembers(params Expression<Func<TDestination, object>>[] memberExpressionTrees);
```

Comment:
I think IgnoreDestinationMembers is a better name, as you can also ignore source members.

Reply:
+1